src/plugins/intel_cpu/src/node.h

```diff
@@ -270,6 +273,23 @@ class Node {
         return !hasEmptyInputTensors();
     }
 
+    /**
+     * Return true if a node can perform preprocessing for an input \idx
+     */
+    virtual bool canPrepInput(size_t idx) const {
+        (void) idx;
+        return false;
+    }
+
+    /**
+     * Require a node to perform \type preprocessing for an input \idx
+     */
+    virtual void prepInput(size_t idx, InputPrepType type) {
+        (void) idx;
+        (void) type;
+        OPENVINO_THROW_NOT_IMPLEMENTED("Input preprocessing is not implemented for node: ", *this);
+    }
```

Comment:
```suggestion
    virtual void prepInput(size_t, InputPrepType) {
        OPENVINO_THROW_NOT_IMPLEMENTED("Input preprocessing is not implemented for node: ", *this);
    }
```
Make unused parameters unnamed then `void` cast should not be required and compiler should report warnings/errors